torchtune/modules/rlhf/utils/_convert_weights.py

```diff
@@ -35,7 +35,7 @@ def mistral_reward_hf_to_tune(
 ) -> Dict[str, torch.Tensor]:
     """
     Convert a state dict from HF's format to torchtune's format, which contains the weights
-    of a Mistral reward model.
+    of a reward model (i.e. a classifier with a single class).
```

Comment:
I guess this confirms that classifier is a case of a reward, and maybe we should swap the current naming?